.env.example

```
@@ -1,32 +1,31 @@
-# Prefix all client variables with NEXT_PUBLIC_
-
 # Road Variables (Prefix ROAD_)
-NEXT_PUBLIC_ROAD_WIDTH
+NEXT_PUBLIC_ROAD_WIDTH=20
+NEXT_PUBLIC_ARROW_SPACING=100
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_ARROW_SPACING key should go before the NEXT_PUBLIC_ROAD_WIDTH key

(UnorderedKey)